app/models/user.rb

```diff
@@ -22,7 +22,11 @@ def janitor?
   end
 
   def admin_for?(organisation)
-    organisation_members&.find(organisation: organisation, role: OrganisationMember::ADMIN_USER).present?
+    organisation_members&.where(organisation: organisation, role: OrganisationMember::ADMIN_USER).present?
```

Comment:
maybe a method called `role_for_org` and then this method says only:
`role == OrganisationMember::ADMIN_USER`